map-storage/src/MapsManager.ts

```diff
@@ -48,7 +49,7 @@ class MapsManager {
         const wamFile = await command.execute();
 
         // Security check: Check that the map is valid after the change (it should be, but better safe than sorry)
-        const map = gameMap.getWam();
+        const map = wamFileMigration.migrate(gameMap.getWam());
```

Comment:
I don't think this is necessary here, since the map was migrated to the latest version when we loaded it. Here, the map should be in memory?